src/VirtoCommerce.WebHooksModule.Data/Utils/WebHookFeedUtils.cs

```diff
@@ -6,9 +6,9 @@ namespace VirtoCommerce.WebHooksModule.Data.Utils
 {
     public static class WebHookFeedUtils
     {
-        public static WebhookFeedEntry CreateErrorEntry(WebhookWorkItem webHookWorkItem, WebhookSendResponse response, string errorMessage)
+        public static WebhookFeedEntry CreateErrorEntry(WebhookWorkItem webHookWorkItem, WebhookSendResponse response, string error = null)
         {
-            return CreateFeedEntry(WebhookFeedEntryType.Error, webHookWorkItem.EventId, response, webHookWorkItem.WebHook, errorMessage);
+            return CreateFeedEntry(WebhookFeedEntryType.Error, webHookWorkItem.EventId, response, webHookWorkItem.WebHook);
```

Comment:
In case of removing `error` param you should probably fill Error in `CreateFeedEntry` from the passed `WebhookSendResponse`, and probably remove error param here too.

Reply:
there is the error in response. it is a redundant field